frontend/public/config.js

```diff
@@ -9,3 +9,4 @@ window.REACT_APP_CROWNLABS_OIDC_PROVIDER_URL = undefined;
 window.REACT_APP_CROWNLABS_OIDC_CLIENT_ID = undefined;
 window.REACT_APP_CROWNLABS_OIDC_CLIENT_SECRET = undefined;
 window.REACT_APP_CROWNLABS_OIDC_REDIRECT_URI = undefined;
+window.REACT_APP_CROWNLABS_GRAPHQL_URL = undefined;
```

Comment:
Since you are adding a constant here, it should also be included in the helm configuration in the following locations (with an example value):
* https://github.com/netgroup-polito/CrownLabs/blob/8e73edf8fbd7fe67e528a508f221fc811dbb52b2/frontend/deploy/frontend-app/values.yaml#L17
* https://github.com/netgroup-polito/CrownLabs/blob/8e73edf8fbd7fe67e528a508f221fc811dbb52b2/deploy/crownlabs/values.yaml#L31
* https://github.com/netgroup-polito/CrownLabs/blob/8e73edf8fbd7fe67e528a508f221fc811dbb52b2/frontend/deploy/frontend-app/templates/configmap.yaml#L9

@GabriFila IIUC, we could completely drop the apiServer entry and replace that with the graphql endpoint, since there are no more direct interactions. Am I correct?